user-management-oidc.md

```diff
@@ -1,73 +1,63 @@
-# User Management (SSO/OIDC)
+# User Management (SSO/OIDC/RBAC)
 
 {% hint style="info" %}
-OIDC capabilities are only officially supported on Kubecost Enterprise plans.
+OIDC and RBAC are only officially supported on Kubecost Enterprise plans.
 {% endhint %}
 
-## Helm configuration
+## Overview of features
 
 The OIDC integration in Kubecost is fulfilled via the `.Values.oidc` configuration parameters in the Helm chart.
 
 ```yaml
+# EXAMPLE CONFIGURATION
+# View setup guides below, for full list of Helm configuration values
 oidc:
-   enabled: true
-   clientID: "" # application/client client_id parameter obtained from provider, used to make requests to server
-   clientSecret: "" # application/client client_secret parameter obtained from provider, used to make requests to server
-   secretName: "kubecost-oidc-secret" # k8s secret where clientSecret will be stored
-   authURL: "https://my.auth.server/authorize" # endpoint for login to auth server
-   loginRedirectURL: "http://my.kubecost.url/model/oidc/authorize" # Kubecost url configured in provider for redirect after authentication
-   discoveryURL: "https://my.auth.server/.well-known/openid-configuration" # url for OIDC endpoint discovery
-#  hostedDomain: "example.com" # optional, blocks access to the auth domain specified in the hd claim of the provider ID token
+  enabled: true
+  clientID: ""
+  clientSecret: ""
+  secretName: "kubecost-oidc-secret"
+  authURL: "https://my.auth.server/authorize"
+  loginRedirectURL: "http://my.kubecost.url/model/oidc/authorize"
+  discoveryURL: "https://my.auth.server/.well-known/openid-configuration"
+  rbac:
+    enabled: false
+    groups:
+      - name: admin
+        enabled: false
+        claimName: "roles"
+        claimValues:
+          - "admin"
+          - "superusers"
+      - name: readonly
+        enabled: false
+        claimName:  "roles"
+        claimValues:
+          - "readonly"
 ```
 
 {% hint style="info" %}
 &#x20;`authURL` may require additional request parameters depending on the provider. Some commonly required parameters are `client_id=***` and `response_type=code`. Please check the provider documentation for more information.
 {% endhint %}
 
+## Setup Guides
```

Comment:
Does this need to be its own section if we're linking to a single guide? This would be better as an indented note or notice.

Reply:
Eventually, I think there will be more guides on this list for other Identity Providers (e.g. Okta, Keycloak, Auth0)